test/unit/formatString-test.js

```diff
@@ -0,0 +1,25 @@
+const test = require(`tape`)
+const format = require(`format`)
+
+test(`format: convert any string to camel case`,  t => {
+  t.plan(3)
+  t.timeoutAfter(100)
+
+  const resultString = `fooBar`
+
+  t.equal(format.camelCase(`foo bar`), resultString)
+  t.equal(format.camelCase(`--foo-bar`), resultString)
+  t.equal(format.camelCase(`__foo_bar__`), resultString)
```

Comment:
Can you also test cases with a single word, e.g. `foobar => foobar` and with more than two cases, `foo-bar-baz__ => fooBarBaz`